src/components/AudioInfo.vue

```diff
@@ -263,14 +263,22 @@
         無効な設定です
       </div>
       <div :class="{ disabled: morphingTargetStyleInfo == undefined }">
-        <span class="text-body1 q-mb-xs"
-          >割合
-          {{
-            morphingRateSlider.state.currentValue.value != undefined
-              ? morphingRateSlider.state.currentValue.value.toFixed(2)
-              : undefined
-          }}</span
-        >
+        <div class="morphing-input">
+          <span class="text-body1 q-mb-xs morphing-label">割合</span>
+          <q-input
+            dense
+            borderless
+            :class="{ disabled: morphingTargetStyleInfo == undefined }"
+            :disable="morphingTargetStyleInfo == undefined"
+            max-length="5"
+            :model-value="
+              morphingRateSlider.state.currentValue.value != null
+                ? morphingRateSlider.state.currentValue.value!.toFixed(2)
+                : null
+            "
+            @change="handleMorphingChange($event)"
+          />
+        </div>
```

Comment:
```suggestion
        <div>
          <q-input
            dense
            borderless
            :class="{ disabled: morphingTargetStyleInfo == undefined }"
            :disable="morphingTargetStyleInfo == undefined"
            max-length="5"
            :model-value="
              morphingRateSlider.state.currentValue.value != undefined
                ? morphingRateSlider.state.currentValue.value!.toFixed(2)
                : morphingRateSlider.qSliderProps.min.value.toFixed(2)
            "
            @change="handleMorphingChange($event)"
          >
            <template #before>
              <span class="text-body1 text-display">割合</span>
            </template>
          </q-input>
        </div>
```

2点ほどの修正を含めたsuggestionをしてみました。

1. 他の実装と合わせる(`template #before`を使う)ことで、新しくCSSを書き足すことなくほかと同じ見た目にできるので、そうしたほうがいいかなと思いました
2. 他の実装で`morphingRateSlider.state.currentValue.value != null`を使っている場所がない(`parameter.slider.state.currentValue.value != undefined`となっていた)ので、それに合わせて`undefined`を使い、三項演算子のelse部分に`null`ではなく`morphingRateSlider.qSliderProps.min.value.toFixed(2)`を使いました。

以上2点、そうしなかった理由などあればコメントいただけると助かります...!(問題なさそうであればそのままsuggestionを取り込んでもらえると....!)